daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -166,8 +166,8 @@ object Speedy {
               CtrlExpr(body)
             case None =>
               throw SpeedyHungry(
-                SResultMissingDefinition(
```

Comment:
The name is different but you always seem to call `SResultNeedPackage` even if the package existed but was just missing the definition here.

Reply:
You are right! I will refactor that. 
Thanks a lot !!

Reply:
Thanks! Looks good to me now :+1: